ui2/src/components/deployment/new/SelectService.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { TableTransfer } from '../../../common/TableTransfer';
+import { Spinner } from '../../../common/Spinner';
+
+export const SelectService = ({ getServices, services, handleBreadcrumbs, getServicesStack, servicesStacks, resetBreadcrumbs, match }) => {
+  useEffect(() => {
+    resetBreadcrumbs();
+    handleBreadcrumbs(`${match.url}`, 'service');
+    getServices();
+    getServicesStack();
```

Comment:
i think the servicesStacks() should return the servicesStacks 
WDYT?

Reply:
Sure!